presto-main/src/test/java/io/prestosql/execution/TestSqlTaskManager.java

```diff
@@ -111,20 +113,19 @@ public void testSimpleQuery()
         try (SqlTaskManager sqlTaskManager = createSqlTaskManager(new TaskManagerConfig())) {
             TaskId taskId = TASK_ID;
             TaskInfo taskInfo = createTask(sqlTaskManager, taskId, ImmutableSet.of(SPLIT), createInitialEmptyOutputBuffers(PARTITIONED).withBuffer(OUT, 0).withNoMoreBufferIds());
-            assertEquals(taskInfo.getTaskStatus().getState(), TaskState.RUNNING);
 
-            taskInfo = sqlTaskManager.getTaskInfo(taskId);
-            assertEquals(taskInfo.getTaskStatus().getState(), TaskState.RUNNING);
+            taskInfo = sqlTaskManager.getTaskInfo(taskId, taskInfo.getTaskStatus().getState()).get(1, TimeUnit.SECONDS);
```

Comment:
this will wait for 1s if `taskInfo.getTaskStatus().getState()` is already FLUSHING. You could use:
```
sqlTaskManager.getTaskInfo(taskId, RUNNING).get(1, TimeUnit.SECONDS);
```
instead